src/func.c

```diff
@@ -1769,11 +1769,15 @@ static void kahanBabuskaNeumaierStep(
 ){
   volatile double s = pSum->rSum;
   volatile double t = s + r;
+  #ifdef FREEBSD_KERNEL
+  //todo: STELIOS
```

Comment:
@stelkasouridis : should you also emit a warning of fail if this code path is traversed?